lucene/test-framework/src/java/org/apache/lucene/search/QueryUtils.java

```diff
@@ -40,9 +39,9 @@
 import org.apache.lucene.util.LuceneTestCase;
 import org.apache.lucene.util.Version;
 
-import static junit.framework.Assert.assertEquals;
-import static junit.framework.Assert.assertFalse;
-import static junit.framework.Assert.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
```

Comment:
Does this change need to be part of this CR?

Reply:
It's not required. I did it because junit.framework.Assert is deprecated.